djangorarapi/fixtures/tokens.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Good work on remembering to seed tokens since it a native django table and not included in the ERD.